framework/spree/utils/get-product-path.ts

```diff
@@ -0,0 +1,7 @@
+import type { ProductSlugAttr } from '../types'
+
+const getProductPath = (partialSpreeProduct: ProductSlugAttr): string => {
```

Comment:
nitpick: I would remove the type annotation to make the code a bit less verbose since TS inference is able to infer this one